### ISSUE TITLE
Primary Button Color in Bootstrap needs to be changed, it does not have the correct color contrast.

### ISSUE THREAD

**Issue by nfaith** [Tim-Bender/CSCI-331-Project#50]
https://webaim.org/resources/contrastchecker/
